package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "metal-myComponent",
+	"version": "0.0.0",
+	"description": "My awesome Metal.js project",
+	"license": "BSD",
+	"repository": "zxdgoal/metal-myComponent",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "src/MyComponent.js",
+	"files": [
+		"src",
+		"test"
+	],
+	"scripts": {
+		"test": "gulp test"
+	},
+	"keywords": [
+		"metal"
+	],
+	"dependencies": {
+    "bootstrap": "^3.3.6",
+		"closure-templates": "20140422.0.0",
+		"metal": "metal/metal.js#master",
+		"metal-component": "metal/metal-component#master",
+		"metal-dom": "metal/metal-dom#master",
+		"metal-events": "metal/metal-events#master",
+		"metal-soy": "metal/metal-soy#master"
+	},
+	"devDependencies": {
+		"gulp": "^3.8.11",
+		"gulp-metal": "^0.14.0"
```

Comment:
I think that you've used an old version of generator-metal to create this, there are a few things that are outdated in some files.

For example, you should be using gulp-metal `^1.0.0` here, instead of `^0.14.0`. There are other similar problems on this file, I'll mention them inline.